test/.default_config

```diff
@@ -1,3 +1,3 @@
-tests.build-image-version=latest
```

Comment:
Why is this changed?

Reply:
should we keep latest for specific version tag?:

Reply:
I think we can keep the latest here, I guess this config is not used for testing of specific release version. @radtriste is it so?

Reply:
nope it is not.
those should be changed if we target another branch but master, like a release branch, which is not the case here